src/states/node.rs

```diff
@@ -2357,38 +2409,34 @@ impl Bootstrapped for Node {
             return result.with_value(Ok(()));
         }
         use routing_table::Authority::*;
-        let sending_names = match routing_msg.src {
-            ClientManager(_) | NaeManager(_) | NodeManager(_) | ManagedNode(_) => {
-                let section = try_ev!(self.peer_mgr
-                    .routing_table()
-                    .get_section(self.name())
-                    .ok_or(RoutingError::RoutingTable(RoutingTableError::NoSuchPeer)),
-                    result
-                );
-                let pub_ids = self.peer_mgr.get_pub_ids(section);
-                vec![SectionList::new(*self.peer_mgr.routing_table().our_prefix(), pub_ids)]
-            }
-            Section(_) => {
-                vec![SectionList::new(*self.peer_mgr.routing_table().our_prefix(), self.peer_mgr
-                    .get_pub_ids(self.peer_mgr.routing_table().our_section()))]
+        let sending_sections = match routing_msg.src {
```

Comment:
If you like, though I generally consider methods a tool for code-reuse more than clean style.